src/sonic-yang-models/yang-events/sonic-events-swss.yang

```diff
@@ -0,0 +1,124 @@
+module sonic-events-swss {
+    namespace "http://github.com/sonic-net/sonic-events-swss";
+    yang-version 1.1;
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+    
+    revision 2022-12-01 {
+        description "SWSS alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC SWSS events";
+
+    container redis-generic {
+        oc-alarm-types:MAJOR
+
+        description "
+            Declares an event for a fatal error encountered by swss.
+            The asic-index of the failing process is the only param.";
+
+        leaf asic_index {
+            type uint8;
+            description "ASIC index in case of multi asic platform";
+            default 0;
+        }
+
+        uses evtcmn:sonic-events-cmn;
+    }
+
+    container if-state {
```

Comment:
Events are declared in container here. Does it mean only one event entry will be verified at a time.

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)